docs/emr/presto-admin-installation-on-emr.rst

```diff
@@ -0,0 +1,32 @@
+=========================
+Presto Admin Installation 
```

Comment:
I guess the question is, what do the docs say about the "default value" of the installer? :) If we put such a note, it looks like a mere suggestion, but in case of EMR, we want them to use online installer by default.